src/authcache.js

```diff
@@ -14,15 +14,15 @@ export class AuthCache {
 
   static get DEFAULT_TTL() { return 300; }
 
-  static _generateKeyHash(username: string, password: string) {
```

Comment:
Because we don't have access to the password when coming from the `allow_publish` and `allow_access` functions and I want to access the cache from them, the hash cannot be based on the password anymore.